### ISSUE TITLE
Can't use useRecoilTransaction with atom which has selector as default value

### ISSUE THREAD

**Issue by smashercosmo** [facebookexperimental/Recoil#1209]
I know that useRecoilTransaction currently doesn't support setting selectors, but it also results in error if you try to set atom's value via useRecoilTransaction if this atom's default value is selector.
Here is codesandbox that demonstrates the issue https://codesandbox.io/s/recursing-lamarr-c6w0q

**Comment by drarmstr** [comment]
Good point, atoms which have a default value that is a selector are implemented via wrapped selectors and thus also not yet supported for transactions.  But, that is the vision of transactions to support this.

**Comment by smashercosmo** [comment]
Alright, got it. Thx for explanation.
